front/src/modules/timesStops/TimesStopsOutput.tsx

```diff
@@ -52,7 +51,7 @@ const TimesStopsOutput = ({
             ? rowData.calculatedArrival !== rowData.arrival.time
             : false;
         const negativeDiffMargins =
-          rowData.diffMargins && Number(rowData.diffMargins?.split(NO_BREAK_SPACE)[0]) < 0;
+          rowData.diffMargins && Number(rowData.diffMargins?.split(' ')[0]) < 0;
```

Comment:
I suggest using `parseInt` instead of` split(' ') + Number` because it directly extracts the integer at the start of the string and ignores any kind of whitespace (including non-breaking spaces). This makes the code simpler and more robust.

```suggestion
          rowData.diffMargins && parseInt(rowData.diffMargins) < 0;
```

Reply:
Done :)